src/Symfony/Component/Lock/NoLock.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Lock;
+
+/**
+ * A non locking lock.
+ *
+ * This is used when no mutex is required for this rate limiter
```

Comment:
This comment should probably be removed or modified as the class is not in the RateLimiter component anymore.